### ISSUE TITLE
showing etherum node connected in mist and it never starts please help 

### ISSUE THREAD

**Issue by manyasurvey** [ethereum/mist#3954]
* [x] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.0.0`
OS & Version: windows/linux/osx
Node version: `geth 0.0.0`
Number of blocks synchronized: 0
```

![capture](https://user-images.githubusercontent.com/32989944/41197914-5ff2580c-6c89-11e8-83d8-fc68c9553e57.PNG)

**Comment by manyasurvey** [comment]
why no one is replying me ): please help me sir

**Comment by BlockWaving** [comment]
got same problem here.  just downloaded Mist 2 days ago.  Is this the new version ryanio talked about, or it's yet to be released?

**Comment by ryanio** [comment]
Hello, we just released a new version (`0.11.0`) that should fix this problem - please give it a try! If you have any new problems that you can reproduce, please create a new issue with the details on how 
we can help. Thanks!
